client/src/test/java/com/kritsit/casetracker/client/domain/services/EditorTest.java

```diff
@@ -37,7 +37,7 @@ public void setUp() {
     }
 
     public void testCreation() {
-        assertTrue(editor instanceof IEditorService);
+        assertNotNull(editor);
```

Comment:
Makes sense. These tests were simply to make the test files actually contain tests when I was creating the classes and then I never bothered to delete them. I suppose it is up for debate whether we should keep them or not, but until that is decided I agree with your change